src/native/libs/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -512,8 +520,26 @@ int32_t CryptoNative_IsSslStateOK(SSL* ssl)
 
 X509* CryptoNative_SslGetPeerCertificate(SSL* ssl)
 {
+    const uint8_t* data = NULL;
+    long len = SSL_get_tlsext_status_ocsp_resp(ssl, &data);
```

Comment:
I don't think OpenSSL supports it, and maybe I am getting a bit ahead of myself, but eventually I think there will be interest in support for multi-stapling.

I wonder if "shove the OCSP staple in the certificate" makes this task any more difficult.

I guess just something to consider as a possible future need.

Reply:
Should that day come, I expect Windows would do it as another cert property, and here we could do a similar thing by either changing our stored value to be `STACK_OF(OCSP_RESPONSE)` or having two different ex_data values.